tests/baselines/reference/newExpressionWithCast.errors.txt

```diff
@@ -1,7 +1,7 @@
 tests/cases/compiler/newExpressionWithCast.ts(3,12): error TS7009: 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.
 tests/cases/compiler/newExpressionWithCast.ts(7,13): error TS2365: Operator '>' cannot be applied to types 'boolean' and 'void'.
 tests/cases/compiler/newExpressionWithCast.ts(7,17): error TS1109: Expression expected.
-tests/cases/compiler/newExpressionWithCast.ts(7,18): error TS2304: Cannot find name 'any'.
+tests/cases/compiler/newExpressionWithCast.ts(7,18): error TS2552: Cannot find name 'any'. Did you mean 'NaN'?

```

Comment:
I decided to fix it here because it'll make the black list smaller; if you legitimately misspell `Boolen` then I think we should give a suggestion.

Not for `NaN` though. Nobody needs more NaN in their code.